charts/actions-runner-controller/Chart.yaml

```
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.12.7
+version: 0.12.8
```

```suggestion
version: 0.12.7
```

Thanks! This almost LGTM, but would you mind reverting the version bump?
The rationale is that the current unreleased version of chart includes changes that requires a new controller release, and the new controller release will take a week or so to be shipped. We don't want to a chart that requires unreleased version of the controller.